my-app/src/constants/constants.service.js

```diff
@@ -0,0 +1,16 @@
+const PROTOCOL = "http";
+const HOST = "localhost";
+const PORT = 3000;
+const VERSION_API = "v1";
+
+const getBaseUrl = () => {
+  return `${PROTOCOL}://${HOST}:${PORT}/api/${VERSION_API}`;
+};
+
+export default Object.assign(
```

Comment:
Why do you  use Object.assign here?

Reply:
If I do it without assign, I get this issue in Eslint -- "Assign object to a variable before exporting as module default  import/no-anonymous-default-export"

Reply:
Then try to fix eslint config, as soon as this is strange behaviour, it should not work like that, i mean why do you need to copy object what you've just created?